app/models/spree/order.rb

```diff
@@ -13,7 +13,7 @@ class Order < ApplicationRecord
     include SetUnusedAddressFields
 
     searchable_attributes :number, :state, :shipment_state, :payment_state, :distributor_id,
-                          :order_cycle_id, :email, :total
+                          :order_cycle_id, :email, :total, :customer_id
```

Comment:
Wow, this increases complexity a lot. Do we need this?

Reply:
without this, the `customer_id_in` params will be ignored by `ransack` and it'll not be possible to filter the line items by customer id.

Reply:
Yes, I was questioning the whole filter by customer feature. I didn't see that in the issue but I probably just overlooked it. I was just pointing out that this feature has a negative impact on performance. That's all. It may not be a problem.